exept/exeptions.py

```diff
@@ -1,23 +1,46 @@
-class SportNotFoundError(Exception):
+class SoftServeException(Exception):
```

Comment:
This should be renamed to something that is project-specific

Any general name will do better that "SoftServe"